VideoApp/VideoApp/Video/Participants/LocalParticipant.swift

```diff
@@ -19,11 +19,12 @@ import TwilioVideo
 
 class LocalParticipant: NSObject, Participant {
     let identity: String
-    var cameraTrack: VideoTrack? { localCameraTrack }
+    let isDominantSpeaker = false
+    let isRemote = false
+    var cameraTrack: VideoTrack? { cameraManager?.track }
```

Comment:
There is a little noise here because I moved some variables. Wanted to list constants before vars for example.